rudder-lang/src/bin/main.rs

```diff
@@ -89,25 +95,33 @@ fn main() {
     // compile should be the default case, so if none of compile / translate is passed -> compile
     let is_compile_default = if !opt.translate { true } else { false };
     let exec_action = if is_compile_default { "compile" } else { "translate" };
-
+    
     logger::set(
         opt.log_level,
         opt.json_log_fmt,
         &exec_action,
     );
-
+    
+    // get default path unless a config path has been specified
```

Comment:
Isn't there teh notion of a default value in structopt so that you don't have to handle that manually ?